### PR TITLE
refactor: replace alphanum-sort with built-in sort

### PR DESCRIPTION
**Pull request by ludofischer**
The order inside selector lists and atrule parameters has no meaning. The reason to sort them is to improve compression. But then the sorting method does not matter as long as it is the same everywhere.

The main changes is that uppercase now comes before lowercase, and numbers are ordered as letters. Also letters come before other symbols like `*` (of course the order inside a single selector still does not change).

Passing this collator to `sort()` gives almost the same result as `alphanum-sort`:

```javascript
new Intl.Collator('en', {sensitivity: 'base', numeric: true, ignorePunctuation: true}).compare
```
But why use a more complicated sort method when the order does not matter anyway?

### CODE REVIEW

File: packages/postcss-minify-selectors/src/__tests__/index.js [hunk 1]
```diff
@@ -96,10 +96,10 @@ test(
 );
 
 test(
-  'should sort using natural sort',
```
Comment (alexander-akait) [inline packages/postcss-minify-selectors/src/__tests__/index.js:99]: I think natural sort has better compression... I think we should check it

Comment (ludofischer) [reply packages/postcss-minify-selectors/src/__tests__/index.js:99]: I am getting the opposite result, natural sort is slightly worse. I am checking with:
```
gzip -6 -c packages/cssnano-preset-default/src/__tests__/integrations/*css | wc -c
```
I get 32411 bytes for natural sort and 32407 bytes for the built-in sort.

Comment (ludofischer) [reply packages/postcss-minify-selectors/src/__tests__/index.js:99]: Gzipping all files to disk gives the same result for both: 356K

**Review by alexander-akait — APPROVED** [review]

**Comment by TrySound** [comment]
😢

**Pull request merged**